README.md

```diff
@@ -1419,6 +1419,7 @@ Please take a quick look at the [contribution guidelines](.github/CONTRIBUTING.m
 * [RazzleDazzle](https://github.com/IFTTT/RazzleDazzle) - A simple keyframe-based animation framework for iOS. Perfect for scrolling app intros.
 * [SwiftyOnboard](https://github.com/juanpablofernandez/SwiftyOnboard) - An iOS framework that allows developers to create beautiful onboarding experiences.
 * [SwiftyWalkthrough](https://github.com/ruipfcosta/SwiftyWalkthrough) - The easiest way to create a great walkthrough experience in your apps.
+* [AwesomeSpotlightView](https://github.com/aleksandrshoshiashvili/AwesomeSpotlightView) - Awesome tool for create tutorial or coach tour
```

Comment:
hi there, thanks for the contribution
can you change the description to "Awesome tool to create tutorial or coach tour."

thanks!